docs/source/changelog.rst

```diff
@@ -8,56 +8,148 @@ Changes in nbconvert
 ---
 `5.4 on Github <https://github.com/jupyter/nbconvert/milestones/5.4>`__
 
-- Document ``--inplace`` command line flag. :ghpull:`839`
-- Don't remove empty cells by default :ghpull:`784`
-- Handle embedded images in html converter :ghpull:`780`
-- If set, use ``nb.metadata.authors`` for LaTeX author line :ghpull:`867`
+Significant Changes
+~~~~~~~~~~~~~~~~~~~
+
+Deprecations
+++++++++++++
+
+Python 3.3 support was dropped. The version of python is no longer common and new versions have many fixes and interface improvements that warrant the change in support.
+
+See :ghpull:`843` for implementation details.
+
+Changes in how we handle metadata
++++++++++++++++++++++++++++++++++
+
+There were a few new metadata fields which are now respected in nbconvert.
+
+``nb.metadata.authors`` metadata attribute will be respected in latex exports. Multiple authors will be added with ``,`` separation against their names.
+
+``nb.metadata.title`` will be respected ahead of ``nb.metadata.name`` for title assignment. This better matches with the notebook format.
+
+``nb.metadata.filename`` will override the default output_filename_template when extracting notebook resources in the ``ExtractOutputPreprocessor``. The attribute is helpful for when you want to consistently fix to a particular output filename, especially when you need to set image filenames for your exports.
+
+The ``raises-exception`` cell tag (``nb.cells[].metadata.tags[raises-exception]``) allows for cell exceptions to not halt execution. The tag is respected in the same way by nbval and other notebook interfaces. ``nb.metadata.allow_errors`` will apply this rule for all cells. This feature is toggleable with the ``force_raise_errors`` configuration option.
```

Comment:
I think we can include most of the details of this in the refactored section on the ExecutePreprocessor. Since it's the next section, I think it's not too bad to mention that these things exist and link below for more details.